core/platform_feature_list.py

```diff
@@ -43,7 +43,7 @@
 # Names of features in dev stage, the corresponding feature flag instances must
 # be in dev stage otherwise it will cause a test error in the backend test.
 DEV_FEATURES_LIST = [
-    params.PARAM_NAMES.dummy_feature
+    params.PARAM_NAMES.dummy_feature.value
```

Comment:
Another comment for this, but can this also just be the enum constant itself rather than its extracted value?

Reply:
I think it should be possible and it is better to keep it as enum constant. Will do it.

Reply:
done